server/app/main.py

```diff
@@ -87,24 +91,37 @@ async def mlmd_push(info: Request):
     print("......................")
     req_info = await info.json()
```

Comment:
Maybe we can add a request validation logic. When body part is empty, we get an internal server error.
![image](https://github.com/user-attachments/assets/a74011d3-e805-41b1-b071-1f07a6894e1c)